examples/bare/src/root.tsx

```diff
@@ -1,12 +1,14 @@
 // @refresh reload
 import { Suspense } from "solid-js";
 import { Body, ErrorBoundary, FileRoutes, Head, Html, Meta, Routes, Scripts } from "solid-start";
+import { Title } from "@solidjs/meta";
```

Comment:
Should the title be imported from '@solidjs/meta' or `solid-start`

Reply:
lets import everything from `solid-start` in the examples